### PR TITLE
[CI] Fix hanging tests [1.3.x]

### PR DESCRIPTION
**Pull request by Tankilevitch**
backport of #3456 & #3457

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -501,7 +501,7 @@ test: clean ## Run mlrun tests
 		--ignore=tests/test_notebooks.py \
 		--ignore=tests/rundb/test_httpdb.py \
 		-rf \
-		tests
+		tests/serving/test_remote.py::test_remote_step
```
Comment (liranbg) [inline Makefile:504]: why

File: .importlinter [hunk 2]
```diff
@@ -0,0 +1,32 @@
+[importlinter]
```
Comment (liranbg) [inline .importlinter:1]: no need

File: Makefile [hunk 3]
```diff
@@ -650,8 +650,13 @@ fmt: ## Format the code (using black and isort)
 	python -m black .
 	python -m isort .
 
+.PHONY: lint-imports
+lint-imports: ## making sure imports dependencies are aligned
+	@echo "Running import linter"
+	lint-imports
```
Comment (liranbg) [inline Makefile:656]: imo can be removed

File: dev-requirements.txt [hunk 4]
```diff
@@ -20,3 +20,4 @@ scikit-learn~=1.0
 lightgbm~=3.0
 xgboost~=1.1
 sqlalchemy_utils~=0.39.0
+import-linter~=1.8
```
Comment (liranbg) [inline dev-requirements.txt:23]: required for 1.3.1?

**Review by liranbg — CHANGES_REQUESTED** [review]

File: dev-requirements.txt (hunk 4 above)
Comment (Tankilevitch) [reply dev-requirements.txt:23]: no, thx

File: Makefile [hunk 5]
```diff
@@ -650,8 +650,13 @@ fmt: ## Format the code (using black and isort)
...
+
 .PHONY: lint
-lint: flake8 fmt-check ## Run lint on the code
+lint: flake8 fmt-check lint-imports ## Run lint on the code
```
Comment (Tankilevitch) [inline Makefile:659]: removed here as well

**Pull request merged**